src/xpk/core/system_characteristics.py

```diff
@@ -55,7 +55,7 @@ class SystemCharacteristics:
   gke_accelerator: str
   gce_machine_type: str
   chips_per_vm: int
-  accelerator_type: AcceleratorType  # type: ignore
+  accelerator_type: int
```

Comment:
Why can't it be `AcceleratorType`? It'd be nice.

BTW is this how we define enums in Python?
```
AcceleratorType = {'TPU': 1, 'GPU': 2, 'CPU': 3}
```